### DISCUSSION TITLE
Swift support in TMs

### DISCUSSION THREAD

**Discussion by troZee** [callstack/react-native-builder-bob#342] [Ideas]
Please add a Swift Support for a TM based on this commit: 

https://github.com/troZee/-swift-support/commit/b7d16b1c8c4dfbb3bdc2e7de11725efe5773e0b8

**Comment by MateWW** [comment]
I believe those are related.
https://github.com/reactwg/react-native-new-architecture/discussions/6#discussioncomment-4389555

**Reply by troZee** [reply]
It refers to a fabric, but we can keep this comment for a visibility
